expr/plan.go

```diff
@@ -98,8 +98,20 @@ func newplan(e *expr, context Context, stable bool, reqs []Req) (GraphiteFunc, [
 		req := NewReq(e.str, context.from, context.to, context.consol)
 		reqs = append(reqs, req)
 		return NewGet(req), reqs, nil
+	} else if e.etype == etFunc && e.str == "seriesByTag" {
```

Comment:
this needs comments about why we need this special case. why can't we just use the etFunc handling below?

Reply:
That's fair. I toyed around and half implemented a version that adds the `seriesByTag` function but I couldn't figure out how to properly integrate it.

The only solution I came up with was checking if `plan.Funcs[0].(type)` was a seriesByTag function.

I'm definitely open to suggestions here, as I do not like this particular section of code (nor the other side that parses it *again*).